backend/src/services/project/project-dal.ts

```diff
@@ -126,13 +126,11 @@ export const projectDALFactory = (db: TDbClient) => {
 
   const findProjectById = async (id: string) => {
     try {
-      const workspaces = await db(TableName.ProjectMembership)
```

Comment:
strange we used to do it like this

Reply:
Yeah I don't see why we're doing it like that. It makes sense to just get the project and joining environments